skyline/analysis/session.py

```diff
@@ -14,13 +14,20 @@
 from skyline.tracking.tracker import Tracker
 from skyline.user_code_utils import user_code_environment
 
+logger = logging.getLogger(__name__)
+
 # habitat imports
-import collections
-import habitat
-from habitat.analysis import SPECIAL_OPERATIONS
-from habitat.profiling.run_time import RunTimeProfiler
+try:
+    import collections
+    import habitat
+    from habitat.analysis import SPECIAL_OPERATIONS
+    from habitat.profiling.run_time import RunTimeProfiler
+
+    habitat_found = True
+except:
```

Comment:
This should only catch ImportErrors instead of all exception types

Reply:
+1